### PR TITLE
chore(container): Enable tests

### PR DESCRIPTION
**Pull request by kakkoyun**

### CODE REVIEW

File: projects/fal/src/fal/logging/isolate.py [hunk 1]
```diff
@@ -17,7 +17,8 @@ class IsolateLogPrinter:
     debug: bool
 
     def __init__(self, debug: bool = False) -> None:
-        self.debug = debug
+        # self.debug = debug
+        self.debug = True
```
Comment (efiop) [inline projects/fal/src/fal/logging/isolate.py:21]:
```suggestion
        self.debug = debug
```

no more need, new isolate is already on shark

File: projects/fal/tests/test_stability.py [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
 
 PACKAGE_NAME = "fall"
 
 
+def git_revision_short_hash() -> str:
+    return (
+        subprocess.check_output(["git", "rev-parse", "--short", "HEAD"])
+        .decode("ascii")
+        .strip()
+    )
```
Comment (efiop) [inline projects/fal/tests/test_stability.py:22]: fal sha doesn't have much to do with this though. Maybe just use a random string to force building?

Comment (kakkoyun) [reply projects/fal/tests/test_stability.py:22]: I didn't want to make it build the environment all the time, actually. I wanted it to be stable within this set of changes.

Comment (efiop) [reply projects/fal/tests/test_stability.py:22]: @kakkoyun this is fragile, doesn't take dirty changes into account and also cwd-dependant.

Fal changes should not matter for the building, right? If we are not testing the building here, then lets just embrace the cached images maybe?

**Comment by efiop** [comment]
```
==> Building the environment
1970-01-01 00:00:00.000 [stderr   ] error: listing workers for Build: failed to list workers: Unavailable: connection error: desc = "transport: authentication handshake failed: tls: failed to verify certificate: x509: certificate has expired or is not yet valid: current time 2024-08-04T23:22:17Z is after 2024-07-25T08:55:23Z"
```

Non GPU builds are failing, need to look further at the backend.

**Pull request merged**